### ISSUE TITLE
Communication to kube-dns broken after removing ipsec container

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#8412]
**Rancher Versions:**
Server: 1.4.3
kubernetes (if applicable): 1.5.2

**Steps to Reproduce:**
- Start kubernetes on 3 hosts
- Deploy one pod
- On the host that has the pod remove the ipsec container
- Wait until the containers recreated

**Results:** 
The pods can reach kube-dns container's ip but dns queries are failing:

```
root@dnstiming-4h5c5:/# dig +short google.com
root@dnstiming-4h5c5:/#

root@dnstiming-4h5c5:/# ping 10.42.172.233
PING 10.42.172.233 (10.42.172.233): 56 data bytes
64 bytes from 10.42.172.233: icmp_seq=0 ttl=62 time=1.221 ms
64 bytes from 10.42.172.233: icmp_seq=1 ttl=62 time=0.978 ms

root@dnstiming-4h5c5:/# ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8): 56 data bytes
64 bytes from 8.8.8.8: icmp_seq=0 ttl=52 time=0.757 ms
64 bytes from 8.8.8.8: icmp_seq=1 ttl=52 time=0.406 ms
```

**Comment by deniseschannon** [comment]
Available in rancher/network-manager:v0.6.0 (network-services template version v0.1.2)

**Comment by galal-hussein** [comment]
Rancher server: v1.5.4-rc1
rancher/net:v0.11.0
rancher/network-manager:v0.6.0
Kubernetes: v1.5.4-rancher1-1

The test pod was able to resolve dns queries after the ipsec container and kubernetes stack restored, it took about 2 minutes.
